src/plugins/template/src/config.cpp

```diff
@@ -51,7 +51,9 @@ ov::Any Configuration::Get(const std::string& name) const {
         return {std::to_string(device_id)};
     } else if (name == CONFIG_KEY(PERF_COUNT)) {
         return {perf_count};
-    } else if (name == ov::template_plugin::throughput_streams || name == CONFIG_KEY(CPU_THROUGHPUT_STREAMS)) {
+    } else if (name == ov::template_plugin::disable_transformations) {
+        return {disable_transformations};
+    } else if (name == CONFIG_KEY(CPU_THROUGHPUT_STREAMS)) {
```

Comment:
```suggestion
    } else if (name == ov::num::streams.name()) {
```
let's use it in tests as well